src/components/Posts/PostInfo/PostInfo.styles.ts

```diff
@@ -5,7 +5,7 @@ export const useStyles = makeStyles(
     root: {
       display: 'flex',
       marginBottom: theme.spacing(8),
-      [theme.breakpoints.down('xs')]: {
+      [theme.breakpoints.down('sm')]: {
```

Comment:
why styles are changing? I don't get that from story 563

Reply:
I slightly updated the moment when we should split the single line into two to avoid dramatic layout breaking. It happened because new date format `hh:mm, dd MM yyyy` much longer than previous `dd.mm.yyyy`

Reply:
understood, thank you for clarification